Gopkg.toml

```diff
@@ -12,4 +12,7 @@
 
 [[constraint]]
   name = "github.com/oracle/oci-go-sdk"
-  version = "v1.0.0"
+  version = "v1.5.0"
+
+[[constraint]]
+  name = "github.com/google/go-cmp"
```

Comment:
I suggest we use https://github.com/stretchr/testify as we're using it on a couple of the other projects already.